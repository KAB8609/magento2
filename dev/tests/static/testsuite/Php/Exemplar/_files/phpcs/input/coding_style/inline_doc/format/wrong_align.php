<?php
/**
 * Short description...
  *
 * License notice...
 *
 * @category    Magento
 *  @package     Magento_Something
 * @copyright   ...
 * @license     ...
 */
class Magento_Something_Model_Block
{
    /**
     * Function that does something
      *
     *@param string $someParam
     * @return string
     */
    public function doSomething($someParam)
    {
        return 'Hello, ' . $someParam;
    }
}
