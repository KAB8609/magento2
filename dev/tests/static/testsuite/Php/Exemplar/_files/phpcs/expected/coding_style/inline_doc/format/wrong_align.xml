<?xml version="1.0" encoding="UTF-8"?>
<config>
    <error line="6"/>
    <error line="13"/>
    <error line="22"/>
    <error line="23"/>
</config>
